app/javascript/components/widgets/forest-change/tree-loss-pct/index.js

```diff
@@ -75,6 +77,11 @@ export default {
     {
       dataset: FOREST_LOSS_DATASET,
       layers: [FOREST_LOSS]
+    },
+    // primary forest
+    {
```

Comment:
You dont want to add this block. By having the forstType: 'primary_forest' you automatically add the layer. Having this adds the layer twice.